doc/rst/language/spec/statements.rst

```diff
@@ -572,13 +572,17 @@ zipper iteration.
 Zipper Iteration
```

Comment:
While there, I suggest `variables` --> `variable(s)` on Line 557. Also, on Line 564, indicate that `in` must be omitted when `index-var-declaration` is omitted.